src/ansible_runner/display_callback/callback/awx_display.py

```diff
@@ -311,7 +295,15 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-Display.display = display_with_context(Display.display)
+for _display_attr in ('banner', 'deprecated', 'display', 'error',
+                      'system_warning', 'verbose', 'warning'):
```

Comment:
The display events "supported" by awx are only:

```
        (0, 'debug', _('Debug'), False),
        (0, 'verbose', _('Verbose'), False),
        (0, 'deprecated', _('Deprecated'), False),
        (0, 'warning', _('Warning'), False),
        (0, 'system_warning', _('System Warning'), False),
        (0, 'error', _('Error'), True),
```

This would potentially apply to `unclassified_display` too.  Although the previous monkeypatching didn't exclude `banner`.